pkg/kubeapiserver/options/authentication.go

```diff
@@ -83,7 +86,10 @@ type WebHookAuthenticationOptions struct {
 }
 
 func NewBuiltInAuthenticationOptions() *BuiltInAuthenticationOptions {
-	return &BuiltInAuthenticationOptions{}
+	return &BuiltInAuthenticationOptions{
+		TokenSuccessCacheTTL: 10 * time.Second,
```

Comment:
Thanks for clarifying. A warning works for me, or just something in the docs for `--authentication-token-webhook-cache-ttl`.

Another option might be to exclude `webhookTokenAuth` from this new cache (leaving them with their own cache), or even to drop `unionAuthTokenHandler` and use a separate cache for each token authorizer (where `--authentication-token-webhook-cache-ttl` would become a flag that set the TTL on that particular cache).